onnxruntime/core/providers/cuda/gpu_data_transfer.cc

```diff
@@ -59,4 +63,79 @@ common::Status GPUDataTransfer::CopyTensor(const Tensor& src, Tensor& dst, int e
 
   return Status::OK();
 }
+
+void PrintTensor(const Tensor& tensor, const std::string& name, bool summary)
```

Comment:
Can we reuse this? [https://github.com/microsoft/onnxruntime/pull/4921/](https://github.com/microsoft/onnxruntime/pull/4921/)

Reply:
I use this code to find numerical issues. However I do not need it for this PR because it turned out to be a frontend issue.